src/pages/GetInvolved/Develop/Develop.js

```diff
@@ -44,6 +45,7 @@ const Develop = props => {
             <UnderlinedLink url={coreNetwork.CTA.url}>{coreNetwork.CTA.name}</UnderlinedLink>
           </div>
         </div>
+        <Invite content={invite} />
```

Comment:
Lets rename this component too be relevant too

Reply:
Don't know how and which are files to change, as style file is named invite.sass